src/quark.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ffi;
+use translate::*;
+
+use std::ffi::CStr;
+
+#[repr(C)]
+pub struct Quark(ffi::GQuark);
```

Comment:
Now also add Eq and PartialEq and Clone and Copy impls :) You can derive them, those are the correct ones

Otherwise all good and perfect

Reply:
I guess Hash and the Ord ones too

Reply:
Ok! ;)